runbotics/runbotics-scheduler/src/mail/mail.service.ts

```diff
@@ -177,6 +177,17 @@ export class MailService {
         }
     }
 
+    public sendUserAcceptMail(userToUpdate: User) {
+        const message = 'Your account has been activated.';
+
+        this.sendMail({
+            to: userToUpdate.email,
+            subject: 'RunBotics - User Activation',
+            content: message,
+            isHtml: false,
+        });
```

Comment:
try to read the user language (EN/PL) and depending on that send the subject and message in appriopriate language